analysis/import/SonarImporter/src/test/java/de/maibornwolff/codecharta/importer/sonar/SonarComponentProjectAdapterTest.java

```diff
@@ -145,4 +146,19 @@ public void should_insert_a_folder_node_from_dir_component() {
         Node actualNode = project.getRootNode().getChildren().get(0);
         assertThat(actualNode.getType(), is(NodeType.Folder));
     }
+
+    @Test
+    public void should_insert_component_from_component_map() {
```

Comment:
test method names are not consistent. Sometimes you use underscores, sometimes you do not. Please pick one